### PR TITLE
package.json, updates

### PR DESCRIPTION
**Pull request by larshp**
add scripts for compiling and testing
add additional packages so the code compiles and it is possible to run linter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,17 @@
   "author": "Sam Jones <sam@decisions.com>",
   "license": "MIT",
   "private": false,
+  "scripts": {
+    "lint": "tslint -c tslint.json 'src/**/*.ts'",
```
Comment (ollwenjones) [inline package.json:11]: This is fine. We should also run lint in a preCompile hook.

File: package.json [hunk 2]
```diff
@@ -7,9 +7,17 @@
...
+    "compile": "tsc"
+  },
   "devDependencies": {
+    "@types/js-cookie": "^2.2.2",
     "jest": "^24.1.0",
     "ts-jest": "^24.0.0",
+    "tslint": "^5.15.0",
+    "tslint-config-prettier": "^1.18.0",
```
Comment (ollwenjones) [inline package.json:19]: tslint-config-prettier is a nice addition.

File: package.json [hunk 3]
```diff
@@ -7,9 +7,17 @@
...
+    "tslint-react": "^4.0.0",
```
Comment (ollwenjones) [inline package.json:20]: I'm not sure about adding tslint-react to this, as my goal was for this to be framework agnostic.

**Comment by ollwenjones** [comment]
@larshp sorry for the slow reply on this. I started it, then put it aside, thinking I needed a little more real use before I could create a tidy helper library. I'm probably in a better position to flesh this out, now, when I find some time.

**Comment by ollwenjones** [comment]
incorporated some of these directly, thanks @larshp

**Comment by larshp** [comment]
cool, changes done in https://github.com/decisions-com/api-helpers/commit/3eadd9c81d9e6d379bd77595916f5bdf3470d3a3, closing

**Pull request closed without merging**